src/components/app/albumsongscontainer/songplayer/song-player.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+
+@inject('songsStore')
+@observer
+class SongPlayer extends Component {
+  constructor(props) {
+    super(props)
+
+    this.props.songsStore.getClickedSong()
+  }
+
+  componentWillReceiveProps() {
```

Comment:
WillReciveProps jest depracated :P Po co to w ogóle jest?

Reply:
co miesiac wszystko jest deprecated :D Czego innego uzyc?

Potrzebne jest do tego jak masz ten player
![image](https://user-images.githubusercontent.com/30021520/53289345-3926a700-3795-11e9-952c-4d479b4eab7a.png)
i wybierasz sobie inna piosenke z albumu to podczytuje ja i musi tez odtwarzacz przeladowac, zeby nowa piosenka byla, a nie leciala stara po prostu

Reply:
Tak, nie dziala :p

Reply:
Meh to zostaw i dodaj komentarz, że to deprecated i trzeba zmienić :P

Reply:
done, zapisalem sobie tez w todo moim